lib/plugins/docker/index.js

```diff
@@ -4,13 +4,25 @@ import preset from '../../pattern-preset';
 
 export default class Docker {
 
+  static clickHandler({ image }) {
```

Comment:
If you agree with my comment at https://github.com/OctoLinker/browser-extension/pull/274/files#r99658736, we should also change `clickHandler` to `resolve` here.